kolibri/plugins/learn/assets/src/my_downloads/views/DownloadsList/index.vue

```diff
@@ -128,25 +131,70 @@
     setup() {
       const { genExternalContentURLBackLinkCurrentPage } = useContentLink();
       const { getLearningActivityIcon } = useLearningActivities();
-
+      const { downloadRequestMap, availableSpace } = useDownloadRequests();
+      const totalPageNumber = ref(0);
+      const store = getCurrentInstance().proxy.$store;
+      const query = computed(() => get(route).query);
+      const route = computed(() => store.state.route);
+      const sort = computed(() => query.value.sort);
+      const pageSizeNumber = computed(() => Number(query.value.page_size || 25));
+      const activityType = computed(() => query.value.activity || 'all');
+      const downloads = ref([]);
+      const sortedFilteredDownloads = () => {
+        let downloadsToDisplay = [];
```

Comment:
Couldn't you just initialize this as `let downloadsToDisplay = Object.values(downloadRequestMap || {});`?

Reply:
I fixed this, but I think after the other changes we made, this was what was causing the duplicate key error that we talked about on slack, so I reverted it